backend/utils/vectors.py

```diff
@@ -67,38 +67,39 @@ def error_callback(exception):
     print("An exception occurred:", exception)
 
 
-def process_batch(batch_ids):
+def process_batch(batch_ids: List[str]):
```

Comment:
Sure about the return type ?

Thanks for typing ☺️

Reply:
yep its a ulid now. We are handling them as strings I think